Makefile

```diff
@@ -74,14 +74,16 @@ curl_local curl_prod:
 	--data "@./data/sample_deal.json" \
 	${CURL_URL}
 
-logfile := logs/src.api.$(shell date +%FT%T).log
-pidfile := logs/server.PID
+LOGDIR  := logs/
+logfile := ${LOGDIR}src.api.$(shell date +%FT%T).log
+pidfile := ${LOGDIR}server.PID
 
 start_local_server: libdds-build stop_local_server
```

Comment:
maybe:
```
-start_local_server: libdds-build stop_local_server
+start_local_server: libdds-build stop_local_server ${LOGDIR}
```

Or the dependency could be declared on a separate line because dependency lists have a bad habit of growing.